packages/integrations/markdoc/package.json

```diff
@@ -27,6 +27,10 @@
       "types": "./dist/extensions/shiki.d.ts",
       "default": "./dist/extensions/shiki.js"
     },
+    "./html": {
+      "types": "./dist/extensions/html/index.d.ts",
+      "default": "./dist/extensions/html/js"
+    },
```

Comment:
It looks like users won't import `@astrojs/markdoc/html` directly. I'd suggest removing this export.